template.tpl

```diff
@@ -502,7 +502,10 @@ const mergeObj = (obj, obj2) => {
 const parseEecObj = prod => {
   return {
     id: prod.id,
-    quantity: prod.quantity
+    quantity: (prod.quantity || 1),
+    title: prod.name,
```

Comment:
mapped properties title, item_price, and category from items name, price, and category properties